modules/developers-guide/pages/cli-reference/dfx-canister.adoc

```diff
@@ -326,7 +328,7 @@ To delete the `+hello_world+` canister, you can run the following command:
 dfx canister delete hello_world
 ----
 
-To delete all of the canisters you have deployed on the `+ic+` Internet Computer network, you can run the following command:
+To delete all of the canisters you have deployed on the `+ic+` {platform}, you can run the following command:
 
 [source,bash]
 ----
```

Comment:
L397: on the {IC} network running locally or on a sub-network that you can access.
=> on the canister execution environment running locally or on the {IC} blockchain.

Reply:
L414: omit "the replica", optionally replace with "the canister execution environment running locally or the {IC} blockchain"